docs/index.md

```diff
@@ -108,6 +108,8 @@ If you cannot find the information that you need in the documentation, help is a
 
     [:octicons-arrow-right-24: FirecREST API](services/firecrest.md)
 
+    [:octicons-arrow-right-24: HPC Console](services/hpc-console.md)
+
```

Comment:
Comment applies here and in `mkdocs.yml`:

FirecREST was moved to "Connecting to Alps". This "service" is very similar, and I feel like it belongs in the same category. Do you think it should be moved alongside the new location for FirecREST?

NB. Please merge latest main (there's a conflict as well; but you'll also see the latest FirecREST changes).